var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -44,4 +49,16 @@ def cmake_args(self):
                 '-DENABLE_CUDA=On',
                 '-DCUDA_TOOLKIT_ROOT_DIR=%s' % (spec['cuda'].prefix)])
 
+        if '+targetopenmp' in spec:
+            options.extend([
+                '-DENABLE_TARGET_OPENMP=On'])
+
+        if '+tbb' in spec:
+            options.extend([
+                '-DENABLE_TBB=On'])
+
+        if '+chai' in spec:
+            options.extend([
+                '-DENABLE_CHAI=On'])
```

Comment:
Switch `extend([...])` to `append(...)`. Also, I would explicitly disable all of these in an else-clause. CMake likes to automatically enable things if it can find the dependencies, leading to non-deterministic builds.